openpgp/packet/aead_config.go

```diff
@@ -15,19 +23,22 @@ type AEADConfig struct {
 
 // Mode returns the AEAD mode of operation.
 func (conf *AEADConfig) Mode() AEADMode {
+	// If no preference is specified use MUST
```

Comment:
Let's spell this out a bit more for those unfamiliar with the spec:
```suggestion
	// If no preference is specified, OCB is used (which is mandatory to implement).
```